### ISSUE TITLE
'High Contrast White' theme Inline Help WCAG AAA contrast compliance

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1034]
**Describe the bug**
The  Inline Help components' trigger button color contrast ratio is not 7:1 or above.

> What version of the Fundamental Library Styles are you using?
v0.10.0-rc.1

**To Reproduce**
1. Go to https://deploy-preview-1025--fundamental-styles.netlify.app/components/inline-help.html
2. Check the trigger button color contrast

**Expected behavior**
If we want WCAG AAA compliance, this contrast ratio should be 7:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81859249-d1838580-9519-11ea-99a2-2ae30068bfc6.png)

![image](https://user-images.githubusercontent.com/43764832/81747592-34fbad80-945d-11ea-8736-e402edc5f967.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
This component is deprecated and no longer in the repo.
